src/index.css

```diff
@@ -1,7 +1,7 @@
 body {
   margin: 0;
-  font-family: Arimo, -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu",
-    "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+  font-family: Arimo, -apple-system, BlinkMacSystemFont, "Open Sans", "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
```

Comment:
Is this in ticket scope? There is another ticket adjusting fonts

Reply:
Also, did we get this from design team (you probably did, but no harm in verifying).

Reply:
@azmainamin Do you mean the font-family? It's based on the figma typography styleguide https://www.figma.com/file/v5HdxDsvO2NzMuZU6CArzF/Resilience-App?node-id=3504%3A6701